brwmaster/task-lister-py
# Finalize setup

## Impact

When this PR gets merged the initial setup for the project is implemented. Also there is a `requirements.txt` file with the dependencies that needs to be installed in order to use this program.

## How to test

If you want to test the code so far. Checkout this branch and make sure that when you run `python -m pip install -r requirements.txt` the correct versions will be installed.
